section2/set9/exercises/03-double-guesses/main.go

```diff
@@ -0,0 +1,84 @@
+// Copyright © 2018 Inanc Gumus
+// Learn Go Programming Course
+// License: https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// For more tutorials  : https://learngoprogramming.com
+// In-person training  : https://www.linkedin.com/in/inancgumus/
+// Follow me on twitter: https://twitter.com/inancgumus
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+// ---------------------------------------------------------
+// EXERCISE: Double Guesses
+//
+//  Let the player guess two numbers instead of one.
+//
+// HINT:
+//  Generate random numbers using the greatest number
+//  among the guessed numbers.
+//
+// EXAMPLES
+//  go run main.go 5 6
+//  Player wins if the random number is either 5 or 6.
+// ---------------------------------------------------------
+
+func main() {
+	const (
+		maxTurns = 5
+		usage    = `Welcome to the Lucky Number Game!
+
+The program will pick %d random numbers.
+Your mission is to guess one of those numbers.
+
+The greater your number is, the harder it gets.
+
+Want to play?`
+	)
+
+	rand.Seed(time.Now().UnixNano())
+
+	args := os.Args[1:]
+
+	if len(args) != 2 {
+		fmt.Printf(usage, maxTurns)
+		fmt.Println()
+		return
+	}
+
+	guess, err := strconv.Atoi(args[0])
+	guess2, err2 := strconv.Atoi(args[0])
```

Comment:
I think it is best to check for err immediately after getting the same.

Reply:
i.e. do not wait for err2

Reply:
code looks cleaner, and both `err != nil` statements will be in one if statement